httpclient5/src/main/java/org/apache/hc/client5/http/ssl/TrustAllStrategy.java

```diff
@@ -34,9 +34,15 @@
 import org.apache.hc.core5.ssl.TrustStrategy;
 
 /**
- * A trust strategy that accepts all certificates as trusted. Verification of
- * all other certificates is done by the trust manager configured in the SSL
- * context.
+ * A trust strategy that accepts all certificates as trusted.
+ *
+ * <h2>Security Warning</h2>
+ * This trust strategy effectivels disables most security features of SSL / TLS,
```

Comment:
@Marcono1234 Could you avoid using the term `most` in `effectively disables most security features of SSL / TLS` because it is absolutely arbitrary? The strategy effectively disables the trust verification but not any other features of TLS. Otherwise looks good to me.

Reply:
Thanks, yes I can change that. Should it just be "disables security features" then, or should it say "disables <ins>some</ins> security features"?

Reply:
@Marcono1234 I would prefer "disables trust verification" but I can live with "disables some security features".